### PR TITLE
Feature/tutorials: ready to go?

### PR DESCRIPTION
**Pull request by paranoya**

### CODE REVIEW

File: src/pst/SSP.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def get_specific_mass_lum_ratio(self, wl_range):
                 mass_to_lum[i, j] = 1/np.mean(self.L_lambda[i, j][pts])
         return mass_to_lum
     
-    def compute_photometry(self, filter_list, z_obs=0.0):
+    def compute_photometry(self, filter_list, z_obs=0.0, verbose=True):
```
Comment (PabloCorcho) [inline src/pst/SSP.py:246]: Can we get rid of this? It feels a bit random having only one method with verbosity...

File: src/pst/SSP.py [hunk 2]
```diff
@@ -257,7 +257,8 @@ def compute_photometry(self, filter_list, z_obs=0.0):
             Array storing the photometry. The dimensions correspond to
             filter, metallicity and age.
         """
-        print("Computing synthetic photometry for SSP model")
+        if verbose:
+            print("Computing synthetic photometry for SSP model")
```
Comment (PabloCorcho) [inline src/pst/SSP.py:261]: And then remove this

**Comment by PabloCorcho** [comment]
I think this is ready to be merged, but I would argue that we should first get rid of the non-informative (and potentially annoying) prints

**Comment by paranoya** [comment]
> I think this is ready to be merged, but I would argue that we should first get rid of the non-informative (and potentially annoying) prints

I have opened issue #45 about logging (I mean `logging` ;^D)

**Comment by PabloCorcho** [comment]
We should make a decision on the verbosity thingy and merge this PR

**Pull request merged**